skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md

```diff
@@ -266,27 +260,30 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -AllowVoicemail
-Enables inbound calls to be routed to voice mail. Valid options are: AlwaysEnabled, AlwaysDisabled, UserOverride.
+### -AutoAnswerEnabledType
+Setting this parameter allows you to enable or disable auto answer for incoming meeting invites on Teams Phones. It is turned off by default. Valid options are Enabled and Disabled. This setting applies only to incoming meeting invites and does not include support for other call types.
 
 ```yaml
-Type: String
+Type: Enum
 Parameter Sets: (All)
 Aliases:
+Applicable: Microsoft Teams
 
 Required: False
 Position: Named
-Default value: None
+Default value: Disabled
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
-### -AllowCloudRecordingForCalls
-Determines whether cloud recording is allowed in a user's 1:1 call. Set this to TRUE to allow the user to be able to record 1:1 calls. Set this to FALSE to prohibit the user from recording 1:1 calls.
+
+### -BusyOnBusyEnabledType
+Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, Disabled. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Note: UserOverride option value is not available for use currently, if set it will be read as setting value to Disabled.
```

Comment:
```suggestion
Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, and Disabled. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Note: The UserOverride option value is not available for use currently, if set it will be read as setting the value to Disabled.
```